substrate/frame/staking-async/src/pallet/impls.rs

```diff
@@ -72,6 +72,18 @@ use sp_runtime::TryRuntimeError;
 const NPOS_MAX_ITERATIONS_COEFFICIENT: u32 = 2;
 
 impl<T: Config> Pallet<T> {
+	/// Returns the minimum required bond for participation, considering validators, nominators,
+	/// and the chain’s existential deposit.
+	///
+	/// This function computes the smallest allowed bond among `MinValidatorBond` and
+	/// `MinNominatorBond`, but ensures it is not below the existential deposit required to keep an
+	/// account alive.
+	pub(crate) fn min_bond() -> BalanceOf<T> {
```

Comment:
```suggestion
	pub(crate) fn min_chilled_bond() -> BalanceOf<T> {
```

and then two more for 

```
fn min_validator_bond
fn min_nominator_bond
```

?

Reply:
@kianenigma Can you sanity check this before I merge

https://github.com/paritytech/polkadot-sdk/pull/8701/commits/81ee0ae158a1a74606b4a0864b365dd58545ae5a#diff-4fde8f6e9b1050fbd0fb764fe2bea06879a3ee0443e476b95ca418d188f22079R89